tools/editor/plugins/resource_preloader_editor_plugin.cpp

```diff
@@ -60,36 +60,40 @@ void ResourcePreloaderEditor::_notification(int p_what) {
 	}
 }
 
-void ResourcePreloaderEditor::_file_load_request(const String& p_path) {
+void ResourcePreloaderEditor::_files_load_request(const Vector<String>& p_path) {
```

Comment:
Maybe use ``p_paths`` instead of ``p_path`` to make it more obvious that it's a vector of paths and not one single path?